commercetools-models/src/main/java/io/sphere/sdk/productdiscounts/ProductDiscountDraftBuilder.java

```diff
@@ -0,0 +1,71 @@
+package io.sphere.sdk.productdiscounts;
+
+import javax.annotation.Nullable;
+
+import io.sphere.sdk.models.Base;
+import io.sphere.sdk.models.Builder;
+import io.sphere.sdk.models.LocalizedString;
+
+public class ProductDiscountDraftBuilder extends Base implements Builder<ProductDiscountDraft> {
+
+    private LocalizedString name;
+    @Nullable
+    private LocalizedString description;
+    private ProductDiscountValue value;
+    private ProductDiscountPredicate predicate;
+    private String sortOrder;
+    private Boolean isActive;
```

Comment:
in http://dev.commercetools.com/http-api-projects-productDiscounts.html#productdiscountdraft 
it is differently called, but in the end only the getter name of ProductDiscountDraft matters, so it can be "active" like in ProductDiscountDraftImpl